runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -475,6 +515,24 @@ static void jitHookInitializeSendTarget(J9HookInterface * * hook, UDATA eventNum
                      TR::Options::getHighCodeCacheOccupancyBCount() :
                      TR::Options::getHighCodeCacheOccupancyCount();
          }
+#if defined(J9VM_OPT_JITSERVER)
+      else if (methodCachedAtServer) // Higher priority than SCC
```

Comment:
I think that this code should not have higher priority than the case when a method is in local SCC, because the JIT will load the method from local SCC if available (it's just faster than the remote AOT cache)